docs/DXIL.rst

```diff
@@ -2721,7 +2721,7 @@ Loads the value from shader input
 Log
 ~~~
 
-Returns log base 2. Note that hlsl log intrinsic returns natural log.
```

Comment:
Are you sure?  I notice that the "Note that hlsl log instrinsic returns natural log" is there to suggest that the "log 2" is surprising, but correct.

Reply:
I think the  docs are correct here and the DXIL log is log_2.

If you compile a shader with `log(x)` you get

```
  %2 = call float @dx.op.unary.f32(i32 23, float %1)  ; Log(value)
  %3 = fmul fast float %2, 0x3FE62E4300000000
```

The `0x3FE62E4300000000` is `0.693147` which is log_e(2).